src/Stratis.Bitcoin/Controllers/NodeController.cs

```diff
@@ -136,6 +142,7 @@ public IActionResult Status()
                 ProtocolVersion = (uint)(this.nodeSettings.ProtocolVersion),
                 Difficulty = GetNetworkDifficulty(this.networkDifficulty)?.Difficulty ?? 0,
                 Agent = this.connectionManager.ConnectionSettings.Agent,
+                ExternalAddress = this.selfEndpointTracker.MyExternalAddress.Address.ToString(),
```

Comment:
Can `MyExternalAddress` ever be null here? I'm, just thinking if we start up the node, the API initializes first and it might not be set? Can you check the code and confirm?

Reply:
No, it ultimately comes from the ExternalEndpoint in the ConnectionManagerSettings, which always falls back to the loopback address if it is null.